brave/brave-ios
# Fix #3358 - Update httpse data files

## Summary of Changes


This pull request fixes #3358 

## Submitter Checklist:


## Test Plan:
1. Navigate to radio.garden and verify the site loads without any redirect loops
2. Navigate to httpse-everywhere.badssl.com and verify that `HTTPS Everywhere` is working

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
